python/lsst/validate/drp/base.py

```diff
@@ -0,0 +1,28 @@
+#!/usr/bin/env python
+
+# LSST Data Management System
+# Copyright 2008-2016 AURA/LSST.
+#
+# This product includes software developed by the
+# LSST Project (http://www.lsst.org/).
+#
+# This program is free software: you can redistribute it and/or modify
+# it under the terms of the GNU General Public License as published by
+# the Free Software Foundation, either version 3 of the License, or
+# (at your option) any later version.
+#
+# This program is distributed in the hope that it will be useful,
+# but WITHOUT ANY WARRANTY; without even the implied warranty of
+# MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+# GNU General Public License for more details.
+#
+# You should have received a copy of the LSST License Statement and
+# the GNU General Public License along with this program.  If not,
+# see <https://www.lsstcorp.org/LegalNotices/>.
+
+from __future__ import print_function, division
+
+
+class ValidateError(Exception):
+    """Base classes for exceptions in validate_drp."""
+    pass
```

Comment:
I'm actually very intentionally creating and raising module-specific errors following the practice suggested at the following URL and in "Effective Python".

https://docs.python.org/2/tutorial/errors.html#user-defined-exceptions

"When creating a module that can raise several distinct errors, a common practice is to create a base class for exceptions defined by that module, and subclass that to create specific exception classes for different error conditions:"

However, I hear you about thinking carefully about whether an error is really distinct to ValidateError or just a general empty-array error, etc.  I will keep this in mind going forward.

E.g.,
```
raise ValidateError("Unknown astrometric test specification: %s" % str(x))
```
seems definitely a specific kind of error.